modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -36,48 +37,170 @@ Showcase apps demonstrate many features of {product-name}. To see these features
 +
 For more information about features, see xref:identity-management.adoc[{keycloak-service} Service]
 
-
 . To demonstrate {unifiedpush-service}, xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
 
-. To demonstrate {sync-service}:
-+
+// tag::excludeDownstream[]
+
 NOTE: You can only demonstrate {sync-service} on Cordova.
 
-.. Provision a PostgreSQL database if there isn't one available.
-... Login to the OpenShift web console, create a new project or use an existing one
-... Click on the `Add to Project` dropdown on the top right, and choose `Browse Catalog`
-... Select `Databases` -> `Postgres` -> `PostgreSQL`, and complete the wizard to start the deployment of the database. Take notes of the values for `PostgreSQL Connection Username`, `PostgreSQL Connection Password` and `PostgreSQL Database Name`. You will need them later.
-.. Provision a new {sync-service} app from the catalog as described in xref:data-sync.adoc#sync-server-openshift[Running Data Sync in OpenShift] with the following parameters:
+// end::excludeDownstream[]
+
+== Demonstrate the {sync-service}
```

Comment:
gerund and avoid definitive article

Reply:
https://github.com/aerogear/mobile-docs/pull/682/commits/5603aac4b52b0bcfd0b0bbe737866d1b653aa7bd